bin/edit_user_role.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * script for editing an existing user
+ */
+import * as parse from 'minimist';
+import * as Knex from 'knex';
+import { Roles } from '../src/auth';
+import { getConfig } from '../config';
+
+process.on('unhandledRejection', (err) => { throw err; });
+
+const { userId, oid, add, rm } = parse(process.argv.slice(2));
```

Comment:
Can you add comments for how this script should be used. Eg what args are passed to it.

Reply:
@eliascodes can this be commented?